Source/ACE/Network/GameEvent/Events/GameEventPlayerDescription.cs

```diff
@@ -11,15 +11,15 @@ public class GameEventPlayerDescription : GameEventMessage
         [Flags]
         private enum DescriptionPropertyFlag
         {
-            None           = 0x0000,
-            PropertyInt32  = 0x0001,
-            PropertyBool   = 0x0002,
+            None = 0x0000,
+            PropertyInt32 = 0x0001,
+            PropertyBool = 0x0002,
             PropertyDouble = 0x0004,
-            PropertyDid    = 0x0008,
+            PropertyDid = 0x0008,
             PropertyString = 0x0010,
-            Position       = 0x0020,
-            PropertyIid    = 0x0040,
-            PropertyInt64  = 0x0080,
+            Position = 0x0020,
+            PropertyIid = 0x0040,
+            PropertyInt64 = 0x0080,
         }
```

Comment:
ya must have hit auto format.   I fixed it.   Found a cool Visual Studio plugin to do that alignment for me.   
https://marketplace.visualstudio.com/items?itemName=NoahRichards.AlignAssignments
and
https://marketplace.visualstudio.com/items?itemName=cpmcgrath.Codealignment

I am trying them both out.